deploy/cloudformation/gomain.go

```diff
@@ -171,3 +188,13 @@ func createStack(stackName string, templatePath string, params map[string]string
 	log.Printf("Created stack %s", *stackOutput.StackId)
 	return nil
 }
+
+func getTemplatePath(deploymentType string, env string) (string, error) {
+	if deploymentType == "" {
```

Comment:
where possible, validations should be in `config.go` under the validation function.